.gitignore

```diff
@@ -12,4 +12,13 @@ yalc.lock
 coverage/
 __generated__
 
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+
+# Barrel files
+packages/client/links
+packages/react/ssg
+packages/server/adapters
+packages/server/observable
+packages/server/rpc
+packages/server/shared
+packages/server/subscription
```

Comment:
I imagine these will get annoying to ignore as it's not automated, I'm fine with just committing them anyway.

Reply:
Yea I initially wanted a glob pattern here to say like, ignore subdirectory of `packages/*` but `src` and `test` but couldn't get that to work. You know what pattern to use?